modules/wms/docs/formats/wfs.md

```diff
@@ -0,0 +1,34 @@
+# WFS Overview
+
+[WFS](https://en.wikipedia.org/wiki/Web_Map_Service) (Web Map Service) is a standardized protocol for serving geo-referenced **map images** over the internet.
```

Comment:
is this the right description? Looks like of WMS.

Reply:
@enersis-pst Copy-pasta. Thanks for noting. Too many similar standards :) Fixing in https://github.com/visgl/loaders.gl/pull/2321/files#diff-c6de9723495ab6f32fa7233f8116b3493e79b6f0566bd17087272e02efa326dbR3